teams.rst

```diff
@@ -0,0 +1,437 @@
+Working in Teams
+================
+
+In some classes, you may need to work on projects in pairs or in teams. This is not a gratuitous
+decision: in practically all contexts, software development is a team sport, and you will
+develop many useful skills by doing these projects with others rather than by working on them
+individually.
+
+However, working with others can be as challenging as it is rewarding. In this page,
+we provide a number of suggestions and tips that can help you make the most out of
+working on a software project with others, particularly in the context of a university
+course.
+
+Choosing your project partner(s)
```

Comment:
Is there a reason this is included in the cs220 version?

Reply:
I think this is a general dev guide for all classes